src/main/java/seedu/address/logic/parser/AddOrderCommandParser.java

```diff
@@ -44,7 +44,7 @@ public AddOrderCommand parse(String args) throws ParseException {
         try {
             index = ParserUtil.parseIndex(argMultimap.getPreamble());
             price = ParserUtil.parsePrice(argMultimap.getValue(PREFIX_PRICE).get());
-            deadline = new Deadline(argMultimap.getValue(PREFIX_BY).get());
+            deadline = ParserUtil.parseDeadline(argMultimap.getValue(PREFIX_BY).get());
             remark = new Remark(argMultimap.getValue(PREFIX_DETAILS).get());
         } catch (NoSuchElementException error) {
```

Comment:
Can you reverse merge `master` for the new format

Reply:
Done, will commit soon